src/julia-parser.scm

```diff
@@ -1162,7 +1162,7 @@
   (let loop ((ex ex))
     (let ((t (peek-token s)))
       (if (or (and space-sensitive (ts:space? s)
-                   (memv t '(#\( #\[ #\{ |'| #\" #\`)))
+                   (memv t '(#\( #\[ #\{ #\' #\" #\`)))
```

Comment:
I'm pretty sure the lexer returns ' as a symbol.

Reply:
Ah, ok, I was a bit confused by that. So the lexer returns `(`, `[`, ... as characters and `'` as a token? At first I thought this was causing the issue, but apparently both work. I can change this back.

Reply:
```suggestion
                   (memv t '(#\( #\[ #\{ |'| #\" #\`)))
```